consul/src/main/scala/io/buoyant/consul/v1/ConsulApi.scala

```diff
@@ -115,23 +116,31 @@ class HealthApi(
       "index" -> blockingIndex,
       "dc" -> datacenter,
       "tag" -> tag,
-      "passing" -> Some("true")
+      "passing" -> Some((statuses == Set(HealthStatus.Passing)).toString)
```

Comment:
`passing=false` means skip filtering. From the (somewhat unclear) [Consul API docs](https://www.consul.io/api/health.html#list-nodes-for-service):

> passing (bool: false) - Specifies that the server should return only nodes with all checks in the passing state. This can be used to avoid additional filtering on the client side.

Reply:
Can we add a comment here indicating that please?